etc/pip_install/toree/toreeapp.py

```diff
@@ -57,6 +59,12 @@ class ToreeInstall(InstallKernelSpec):
     spark_home = Unicode(os.getenv(SPARK_HOME, '/usr/local/spark'), config=True,
         help='''Specify where the spark files can be found.'''
     )
+    hadoop_conf_dir = Unicode(os.getenv(HADOOP_CONF_DIR, '/usr/local/hadoop'), config=True,
+        help='''Specify where the hadoop config files can be found.'''
+    )
+    spark_conf_dir = Unicode(os.getenv(SPARK_CONF_DIR, '/usr/local/spark'), config=True,
+        help='''Specify where the spark config files can be found.'''
```

Comment:
Should the default value actually be /usr/local/spark/conf ?

Reply:
sure, it has to be the place where `spark-env.sh` is found.